tool/tctl/common/auth_command.go

```diff
@@ -737,6 +737,15 @@ func (a *AuthCommand) checkProxyAddr(clusterAPI auth.ClientI) error {
 		return nil
 	}
 	if a.proxyAddr != "" {
+		// User set --proxy. Validate it and set its scheme to https in case it was omitted.
+		u, err := url.Parse(a.proxyAddr)
+		if err != nil {
+			return trace.WrapWithMessage(err, "Specified --proxy URL is invalid")
+		}
+		if u.Scheme == "" {
+			u.Scheme = "https"
+		}
+		a.proxyAddr = u.String()
```

Comment:
Probably minimal gain but you could avoid the call to `u.String()` in the event the scheme is present. Since we are now parsing the user input should we also limit the provided scheme to only be `https` or `http`?

Reply:
Thanks for the review @rosstimothy. Tried to do this in 5bf4fdcb25415f0d30eb5da630801230041dbfa8.